node/task.ts

```diff
@@ -1157,6 +1159,27 @@ export function globFirst(pattern: string): string {
     return matches[0];
 }
 
+export function findFiles(includePattern: string[], excludePattern: string[], workingDir?: string): string[] {
```

Comment:
If exclude patterns is to be pulled into the lib, then tl.match should have an overload. At that point, this function would become a thin wrapper over tl.find and tl.match.

I like the idea of a wrapper function so caller doesn't need to setup the options (specify matchbase:true and if-windows case-insensitive:true). However, I recently added fixes and unit tests for find and the function you are adding here does not expose the same options (and likely behaves differently too). We should drive consistency, not add different find/globbing functionality.